### DISCUSSION TITLE
v3.6.3

### DISCUSSION THREAD

**Discussion by danielroe** [nuxt/nuxt#22147] [Announcements]
> 3.6.3 is the next patch release, including a number of fixes. It's anticipated this will be the last patch release before 3.7.

## ✅ Upgrading

As usual, our recommendation for upgrading is to run:

```sh
nuxi upgrade --force
```

This will refresh your lockfile as well, and ensures that you pull in updates from other dependencies that Nuxt relies on, particularly in the unjs ecosystem.

## 👉 Changelog

[compare changes](https://github.com/nuxt/nuxt/compare/v3.6.2...v3.6.3)

### 🔥 Performance

- **nuxt:** Prepopulate island payloads from rendered html ([#22049](https://github.com/nuxt/nuxt/pull/22049))

### 🩹 Fixes

- **kit:** Module compatibility not working with module names ([6cabe69ce](https://github.com/nuxt/nuxt/commit/6cabe69ce))
- **schema:** Correct type of `unctx` options ([4e32e70bb](https://github.com/nuxt/nuxt/commit/4e32e70bb))
- **vite:** Pass modulesDir to `isExternal` ([#21966](https://github.com/nuxt/nuxt/pull/21966))
- **nuxt:** Add pages in correct hook when generating ([#22088](https://github.com/nuxt/nuxt/pull/22088))
- **test-utils:** Export from core playwright package ([#22122](https://github.com/nuxt/nuxt/pull/22122))
- **nuxt:** Skip view transitions where there is no route change ([#22140](https://github.com/nuxt/nuxt/pull/22140))
- **nuxt:** Skip scanning components that do not produce a name ([#22074](https://github.com/nuxt/nuxt/pull/22074))
- **cli:** Add aliases to monorepo paths to tsconfig includes ([#21997](https://github.com/nuxt/nuxt/pull/21997))
- **cli:** Allow non-existent `experimental` option ([0643d4315](https://github.com/nuxt/nuxt/commit/0643d4315))

### 💅 Refactors

- **cli,schema:** Add `bundler` module resolution flag ([#22142](https://github.com/nuxt/nuxt/pull/22142))

### 📖 Documentation

- Fix accessing custom props from child pages ([#21977](https://github.com/nuxt/nuxt/pull/21977))
- Add leading slash ([3bae61507](https://github.com/nuxt/nuxt/commit/3bae61507))
- Add more internal links ([#22013](https://github.com/nuxt/nuxt/pull/22013))
- Add space in list ([#22032](https://github.com/nuxt/nuxt/pull/22032))
- Add details and components folder to auto imports introduction ([#22062](https://github.com/nuxt/nuxt/pull/22062))
- Use sfc syntax in example ([#22073](https://github.com/nuxt/nuxt/pull/22073))
- Prefix internal links with `/` ([#22118](https://github.com/nuxt/nuxt/pull/22118))
- Fix typos ([#22125](https://github.com/nuxt/nuxt/pull/22125))
- Update link to router options ([#22131](https://github.com/nuxt/nuxt/pull/22131))

### 🏡 Chore

- Add explicit vue dependency in fixtures ([c10a62405](https://github.com/nuxt/nuxt/commit/c10a62405))
- Update lockfile ([6481162f4](https://github.com/nuxt/nuxt/commit/6481162f4))
- Add explicit type dependencies ([d590c85c3](https://github.com/nuxt/nuxt/commit/d590c85c3))
- Add volar to workspace recommendations ([0c826ee27](https://github.com/nuxt/nuxt/commit/0c826ee27))
- Add eslint schema ([4d7df49eb](https://github.com/nuxt/nuxt/commit/4d7df49eb))

### ❤️  Contributors

- Daniel Roe (@danielroe)
- Samuel Burkhard (@samydoesit)
- 云游君 (@YunYouJun)
- Jianqi Pan (@Jannchie)
- Luke Pinion (@lwpinion)
- Fanny (@FannyGautierr)
- Hebilicious (@Hebilicious)
- Eduardo San Martin Morote (@posva)
- Eduardo Santos De Brito (@EduSantosBrito)
- Nurassyl (@Nura-21)
- Zakir Hossen (@devzakir)
- Harlan Wilton (@harlan-zw)

<hr /><em>This discussion was created from the release <a href='https://github.com/nuxt/nuxt/releases/tag/v3.6.3'>v3.6.3</a>.</em>

**Comment by jd1378** [comment]
After upgrading from 3.6.2 to 3.6.3, `nuxt prepare` command fails with:
```
 ERROR  nuxt.options.typescript?.tsConfig?.compilerOptions?.moduleResolution?.toLowercase is not a function    
```
my nuxt config about the error part:
```ts
export default defineNuxtConfig({
  // ...
  typescript: {
    shim: false,
    tsConfig: {
      compilerOptions: {
        moduleResolution: 'bundler', // If I comment this line, the error does not happen anymore
        strictNullChecks: true,
        strict: true,
      },
    },
  },
 }
```
if I comment `moduleResolution`, the error does not happen anymore

**Reply by jd1378** [reply]
well it seems https://github.com/nuxt/nuxt/commit/fed5868184c6ec7256c114cb804253b8deacdf64 has fixed it already, have to skip this version and wait for 3.7.0 I guess
